src/server/core/connection.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import logger from 'logger/logger';
+
+const instance = axios.create({
+  baseURL: 'http://localhost:3000/api', // TODO(asclines): Move this to env
+  timeout: 5000,
```

Comment:
Is this a single instance for the entire app? If so, the timeout might be something of a problem